README.md

```diff
@@ -1,6 +1,121 @@
+  - Authentication Types:
+    - Fence Token
+- Gabriella Miller Kids First DRC
+  - Authentication Types:
+    - Fence Token
+- Passport Test Provider
+  - Authentication Types:
+    - Passports
+    - Fence Tokens
+
+## Usage
+To resolve a DRS URL, perform an HTTP `POST` to the `/api/v4/drs/resolve`.
+The content-type of your request should be application/json with the content/body of your request encoded accordingly.
+
+Request bodies should look like 
+```json
+{
+  "url": "string",
+  "fields": ["string"]
+}
+```
+where `url` is the DRS URL to resolve and `fields` is any of
+```text
+accessUrl
+bondProvider
+bucket
+contentType
+fileName
+googleServiceAccount
+gsUri
+hashes
+localizationPath
+name
+size
+timeCreated
+timeUpdated
+```
+
+By default, the response will include 
+```text
+bucket
+contentType
+fileName
+gsUri
+hashes
+localizationPath
+name
+size
+timeCreated
+timeUpdated
+googleServiceAccount
+```
+but this can be changed by providing values in the `fields` array.
+
+## Architecture
+DrsHub is a Java 17 Spring Boot application running in Kubernetes. As it simply resolves urls and doesn't have any state, it has no database. For developer convenience, a Swagger UI is provided.
+
+Some architecture diagrams can be found in [LucidChart](https://lucid.app/documents#/documents?folder_id=297026717)
+
+## Development
+
+### Setup
+Install Java 17 SDK from your preferred provider. A common way to install and manage different JDK versions is to use [sdkman](https://sdkman.io/). If developing in IntelliJ, you can just configure the Project SDK to use Java 17.
+
+You must use [git-secrets](https://github.com/awslabs/git-secrets). You should be doing this anyway for all of your repositories.
+
+Before running anything, make sure to run `./render_configs <ENV>` to render secrets locally. The default `<ENV>` is `dev`, which is what you should use for local running and testing.
+
+DrsHub uses Gradle as a build tool. Some common Gradle commands you may want to run are
+```shell
+./gradlew generateSwaggerCode # Generate Swagger code for models and Swagger UI
+./gradlew bootRun # Run DrsHub locally (Swagger UI at localhost:8080)
+./gradlew test # Run the unit tests
+./gradlew jib # Build the DrsHub Docker image
+```
+
+### Run Integration Tests
+DrsHub uses [TestRunner](https://github.com/DataBiosphere/terra-test-runner) to run its integration tests.
+To run the integration test suite, run
+```shell
+./gradlew runTest --args="suites/FullIntegration.json /tmp/test-results"
+```
+Adding `--stacktrace` can give you more debugging information, if needed.
+
+### Logging
 By default, DrsHub will emit logs in the Stackdriver JSON format. 
-To disable this behavior for local development, add `DRSHUB_LOG_APPENDER=Console-Standard` to your environment when running DrsHub.
\ No newline at end of file
+To disable this behavior for local development, add `DRSHUB_LOG_APPENDER=Console-Standard` to your environment when running DrsHub.
+
+## Deployment
```

Comment:
Can we add instructions for how to manually perform a deployment of the latest version and/or a specific commit or branch?

Reply:
Unsure if that should go in the public README or the private Playbook